modules/history.js

```diff
@@ -198,7 +198,7 @@ History.prototype.validateEntryField = function(field, value) {
 History.prototype.set = function(fieldNameOrHistoryObject, historyValue) {
   // Is the first argument a field name or a full history object?
   if (typeof fieldNameOrHistoryObject === 'string') {
-    if (this.validateEntry(fieldNameOrHistoryObject, historyValue)) {
+    if (this.validateEntry(historyValue)) {
```

Comment:
The validateEntry is written it only takes one argument and that argument is an object. See commit: aaac6271f8e9578766a5d4e48f69dce7850ed88c